addons/hellfire/CfgMagazines.hpp

```diff
@@ -24,7 +24,7 @@ class CfgMagazines {
         count = 1;
         mass = 85;
         pylonWeapon = QGVAR(launcher);
-        hardpoints[] = {"B_MISSILE_PYLON", "SCALPEL_1RND_EJECTOR", "B_ASRRAM_EJECTOR", "UNI_SCALPEL", "CUP_NATO_HELO_SMALL", "CUP_NATO_HELO_LARGE", "RHS_HP_MELB"};
```

Comment:
Whats going on here?

Reply:
wat? wtf

Reply:
Fixed. And I just skimmed over all changes again and it looks like only my changes are left.

Reply:
make sure to push the changes

Reply:
XD I did push... But I didn't commit before 
Pushed now.. I will go to bed now..